### ISSUE TITLE
FYI, OpenPGP is *not* covered by the GPL in any way

### ISSUE THREAD

**Issue by singpolyma** [jasonhinkle/php-gpg#22]
Some implementations of it are, but that does not affect alternate implementations of the same spec (unless you're basing your code on GPL'd implementation code).

**Comment by singpolyma** [comment]
That said, I love the GPL so feel free to still use that for your license ;)

**Comment by DanielRuf** [comment]
I am just a contributor. Can you show us the relevant parts? The code from the company which created this library licensd it under the GPL.

**Comment by singpolyma** [comment]
Sure, if the library creator puts it under the GPL, that's fine. But the readme implies that because gnupg is GPL'd that would somehow affect this project, which it does not (unless you're directly adapting code from there).

**Comment by DanielRuf** [comment]
Ok, thanks for the heas u, will fx this in the readme and mention this.

@jasonhinkle how could we write / reword this?

**Comment by jasonhinkle** [comment]
Well as the readme does mention I'd like to release under a more permissive license but the original PHP-GPG was commissioned as an exact port of GPG C code so to be safe I assumed we should be GPL also.

I wish I had the exact details and version of GPG we used but it's been a long time (way before GitHub existed). The original coder (who I hired and paid for doing the initial port) has totally vanished and I just tried to keep this alive over the years with tweaks and bug fixes.
